packages/jest/src/migrations/update-12-6-0/update-base-jest-config.ts

```diff
@@ -0,0 +1,51 @@
+import { formatFiles, Tree } from '@nrwl/devkit';
+import { jestConfigObject } from '../../utils/config/functions';
+import { getJestProjects } from '../../utils/config/get-jest-projects';
+
+const REPLACE_TOKEN = `$$REPLACE__ME$$`;
+
+function determineUncoveredJestProjects(existingProjects: string[]) {
+  const coveredJestProjects = (getJestProjects() as string[]).reduce(
+    (acc, key) => {
+      acc[key] = true;
+      return acc;
+    },
+    {}
+  );
+  return existingProjects.filter((project) => !coveredJestProjects[project]);
+}
+
+function determineProjectsValue(uncoveredJestProjects: string[]): string {
+  if (!uncoveredJestProjects.length) {
+    return `getJestProjects()`;
+  }
+  return `[...getJestProjects(), ${uncoveredJestProjects.map(
+    (projectName) => `'${projectName}', `
+  )}]`;
+}
+
+function updateBaseJestConfig(
+  tree: Tree,
+  baseJestConfigPath = 'jest.config.js'
+) {
+  const currentConfig = jestConfigObject(tree, baseJestConfigPath);
+  const uncoveredJestProjects = determineUncoveredJestProjects(
```

Comment:
@Cammisuli Can you help write this migration using the AST property removers? I'm not sure if we have a util for removing certain items from an array :thinking: 

I'm thinking:

1. Get the list of projects using `getJestProjects()`
2. Remove each of those projects from the list of projects
3. Prepend `getJestProjects()`

This should naturally leave projects which weren't in `getJestProjects()` still in the config.

The result should be

```ts
module.exports = {
  projects: [
    ...getJestProjects(),
    '<rootDir>/uncovered-project'
  ]
};